src/main/kotlin/uk/gov/justice/digital/hmpps/supportadditionalneedsapi/domain/entity/ElspPlanEntity.kt

```diff
@@ -76,8 +76,8 @@ data class ElspPlanEntity(
   var createdBy: String? = null,
 
   @CreationTimestamp
-  @Column(updatable = false)
-  var createdAt: Instant? = null,
```

Comment:
Because the tests rely on a plan being created in the past I've had to remove this annotation and set a default value which is a shame but it's the only way I can see to do this.